cypress/support/generators/index.js

```diff
@@ -0,0 +1,29 @@
+import strings from './naughtyStrings.json'
+
+export const sampleArr = arr => arr[Math.floor(Math.random() * arr.length)];
+
+export const genString = (len, srcArr) => {
+    const generate = (len, acc) => {
+        if (acc.length >= len) {
+            return acc.substring(0, len);
+        }
+
+        return generate(len, `${acc}${sampleArr(srcArr)}`);
+    };
+
+    return generate(len, sampleArr(srcArr));
+};
+
+export const genNaughtyString = (len) => {
+    const generate = (len, acc) => {
```

Comment:
arent these `const generate` utils nearly the same in `genString` and `genNaughtyString`?  Can we make one utility to use in both